README.md

```diff
@@ -267,6 +267,18 @@ We at least will update the blog post to indicate that it is not expected to
 work with the latest version of the code.
 
 
+You can also manually install by:
+
+Following the instructions in the Docker file
+Dockerfile: [`docker/seahorn-builder.Dockerfile`](docker/seahorn-builder.Dockerfile).
+
+If using LLVM-14 from apt.LLVM.org (version with package bugs)
+Fist running the below commnad to fix an issue with LLVM-14 and libpolly-14
```

Comment:
This will not work. Give complete sequence of commands. It would be something along the line of
```
./llvm.sh 14 && apt download ... && dpkg ... 
```
First install `clang-14` and the rest of llvm, then force install missing `libpolly` library.